WordPress/Classes/Models/PostStatus.h

```diff
@@ -0,0 +1,13 @@
+#import <CoreData/CoreData.h>
+#import "Blog.h"
+
+@interface PostStatus : NSManagedObject
```

Comment:
This is a very nice addition... one thing that's missing though is documentation on this class and its members (even though the naming of the class and its members is pretty straightforward).